src/renderers/webgl/WebGLBackground.js

```diff
@@ -189,7 +191,13 @@ function WebGLBackground( renderer, cubemaps, state, objects, alpha, premultipli
 
 	function setClear( color, alpha ) {
 
-		state.buffers.color.setClear( color.r, color.g, color.b, alpha, premultipliedAlpha );
+		const outputColorSpace = renderer.outputEncoding === sRGBEncoding
```

Comment:
Maybe with the switch of `renderer.outputEncoding` to `renderer.outputColorSpace` it would also make more sense to have as its value `LinearSRGBColorSpace` and `SRGBColorSpace` instead or `LinearEncoding` and `sRGBEncoding`? Then this line can be removed.

Reply:
#23936 would simplify this PR, yes. 👍